portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Ariana Zhu | Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="content">
+      <h1>Ariana Zhu</h1>
+      <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="projects.html">Projects</a></li>
+          <li><a href="gallery.html" class="active">Gallery</a></li>
+          <li><a href="contact.html">Contact</a></li>
+      </ul>
+    </div>
+
+    <div class="content">
+        <h2>Photos from some of my favorite places I've visited. I'd love to explore new cities around the world (once all this is over)!</h2>
+    </div>
+
+    <div class="content">
+      <div class="row">
+          <div class="column">
+              <div class="container">
+                <img src="/images/provence_view.jpg" class="image">
```

Comment:
As a general rule, it is good to associate alt text with images. That improves accessibility by giving screen reader tools something to announce when they encounter an image. Makes the web page work for blind users. It also shows something a little better than the broken image view if for some reason the browser can't load or find the image.

So I would add something like  alt="Provence, France".
Since there is also a caption by the image, I think that very short alt text is all that is needed.